diktat-rules/src/test/resources/test/paragraph2/header/CopyrightDifferentYearExpected2.kt

```diff
@@ -0,0 +1,19 @@
+/*
+    Copyright (c) My Company., Ltd. 2021. All rights reserved.
```

Comment:
Should we fix it to `2003-2021` or just `2021`?

Reply:
As I see it now, we generally have two different formats of copyrights dates - with one single date like in this test, and with 2 dates: `startYear - endYear` and this style is defined in configuration of the rule.

So, I think if user set configuration with single year, it should always use latest date and we probably shouldn't mixing the styles. If he would like to show the continuation of development, he can choose another pattern

Reply:
Okay, makes sense